src/Directory.Build.targets

```diff
@@ -36,6 +36,22 @@
     
   </Target>
 
+  <Target Name="AddConflictResolutionAssetsForTests"
+          BeforeTargets="_HandlePackageFileConflicts"
+          Condition="'$(IsTestProject)' == 'true' and '@(DefaultReferenceExclusions)' != ''">
+    <ItemGroup>
+      <Reference Include="@(DefaultReferenceExclusions->'$(RefPath)%(Identity).dll')" />
```

Comment:
What was this for?  It seems like you're trying to coflict resolve against exclusions, but then you just remove them regradless.  Why not instead just remove any exclusions after?

Reply:
That's a good suggestion. Remove the reference after the conflict resolution is done. For that I need to touch arcade, let me do that in a subsequent PR, whenever I touch CoreFxTesting again.